testData/solutions/win32.json5

```diff
@@ -1208,14 +1289,82 @@
                     "path": "Software\\Microsoft\\Narrator\\NoRoam",
                     "dataTypes": {
                         "SpeechVolume": "REG_DWORD",
+                        "SpeechVoice": "REG_SZ",
                         "ShowKeyboardIntroduction": "REG_DWORD",
-                        "ShowBrowserSelection": "REG_DWORD"
+                        "ShowBrowserSelection": "REG_DWORD",
+                        "ContextVerbosityLevel": "REG_DWORD",
+                        "RenderContextBeforeElement": "REG_DWORD",
+                        "DuckAudio": "REG_DWORD"
                     }
                 },
                 "supportedSettings": {
-                    "SpeechVolume": {},
+                    "SpeechVolume": {
+                        "schema": {
+                            "title": "Speech volume",
+                            "description": "Changes the Narrator voice volume",
+                            "type": "number",
+                            "range": "0.01..1"
+                        }
+                    },
+                    "SpeechVoice": {
+                        "schema": {
+                            "title": "Speech voice",
+                            "description": "Select which voice is going to be used for Narrator",
+                            "type": "enum",
+                            "default": "Microsoft David - English (United States)",
+                            "enum": [
+                                "Microsoft David - English (United States)",
+                                "Microsoft Zira - English (United States)",
+                                "Microsoft Mark - English (United States)",
+                                "Microsoft David Desktop",
+                                "Microsoft Zira Desktop"
+                            ],
+                            "outputFormat": [
+                                "Microsoft David - English (United States)",
+                                "Microsoft Zira - English (United States)",
+                                "Microsoft Mark - English (United States)",
+                                "Microsoft David Desktop",
+                                "Microsoft Zira Desktop"
+                            ]
+                        }
+                    },
                     "ShowKeyboardIntroduction": {},
-                    "ShowBrowserSelection": {}
+                    "ShowBrowserSelection": {},
+                    "ContextVerbosityLevel": {
+                        "schema": {
+                            "title": "Context Verbosity Level",
+                            "description": "Changes the amount of contextual reading for controls and buttons",
+                            "type": "enum",
+                            "default": 0,
+                            "enum": [0, 1, 2, 3, 4, 5],
+                            "outputFormat": [
```

Comment:
These should read "enumLabels" rather than "outputFormat" as I understand the implementation of @the-t-in-rtf